pom.xml

```diff
@@ -30,6 +30,9 @@
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-java</artifactId>
       <version>4.18.1</version>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>4.18.1</version>
```

Comment:
**Security Issue**: Duplicate dependency declaration for selenium-java. This redundancy could lead to confusion and maintenance issues, and potentially mask a different version being included elsewhere, affecting the application's behavior or security. <br> **Fix**: Remove the duplicate selenium-java dependency declaration to ensure clarity and maintainability of the project dependencies. <br> **Code Suggestion**: 
 ```
 -      <groupId>org.seleniumhq.selenium</groupId>
 -      <artifactId>selenium-java</artifactId>
 -      <version>4.18.1</version>
 ```